test/e2e/specs/media/media__upload.ts

```diff
@@ -30,9 +31,10 @@ describe( DataHelper.createSuiteTitle( 'Media: Upload' ), () => {
 
 	// Parametrized test.
 	describe.each`
-		siteType      | accountName
-		${ 'Simple' } | ${ 'defaultUser' }
-		${ 'Atomic' } | ${ 'eCommerceUser' }
+		siteType       | accountName
+		${ 'Simple' }  | ${ 'defaultUser' }
+		${ 'Atomic' }  | ${ 'eCommerceUser' }
+		${ 'Jetpack' } | ${ 'jetpackUser' }
```

Comment:
It was decided to simply add `Jetpack` to the list of parametrized test here for simplicity.

Reply:
Ditto for https://github.com/Automattic/wp-calypso/pull/62324/files#r839027390